### PR TITLE
Updates to the bot's script

### PR DESCRIPTION
**Pull request by chriswilk**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -318,8 +320,20 @@ function buildGenericAttachment(elements) {
...
 }
 
-const NO_DATA_TEXT = "Sorry, I've not found you anything for that."
-const HELP_TEXT = "Type 'suggest' to see 3 inspiring recipes or type the name of a cuisine, course, ingredient, dish or dietary requirement (e.g. vegan or gluten free) to search for a recipe. Easy!"
+// Not used yet - const FEEDBACK_TEXT = "I'd love to know what you make of my sous chef skills. Could you spare a few moments to give some feedback?"
+// Not used yet - const NOT_SMART_ENOUGH_TEXT = "Sorry, I'm not smart enough for complex commands. Could you rephrase your request?"
+const NO_DATA_TEXT = "Sorry, I couldn't source anything for that. What kind of meal do you fancy eating? For example, you could type 'pasta' or 'breakfast', or try 'chicken curry' or 'vegetarian'."
+const HELP_TEXT = "Type 'suggest' to get three popular recipes or type the name of a cuisine, course, ingredient, dish or diet (e.g. 'vegan' or 'gluten free'). Easy!"
+
+const WELCOME_TEXT_1 = "I'm a virtual sous-chef, created by The Guardian to help you with your cooking. I can suggest great recipes that you might like to cook."
```
Comment (alastairjardine) [inline index.js]: Guardian style is "**t**he Guardian" when mentioned in a context like this

File: index.js [hunk 2]
```diff
@@ -318,8 +320,20 @@ function buildGenericAttachment(elements) {
...
+const WELCOME_TEXT_3 = "To get started, type ‘suggest’ and I’ll inspire you with three popular recipes."
+
+const POST_SUGGEST_TEXT_1 = "Now you get the idea. You can ask me to find recipes by the name of a cuisine, course, ingredient, dish or dietary requirement. I will then try find you something great from my database of recipes"
+const POST_SUGGEST_TEXT_2 = "What would you like me to cook up?"
+
+const QA_HELPFUL_RESULT_TEXT_YES = "Yes"
+const QA_HELPFUL_RESULT_TEXT_NO = "No, find me more"
 
 const noDataMessages = [
   "Sorry, I've not found you anything for that."
```
Comment (alastairjardine) [inline index.js]: Would "I can't find" be better then "I've not found"? Keeps it in the present tense

File: index.js [hunk 3]
```diff
@@ -318,8 +320,20 @@ function buildGenericAttachment(elements) {
   }
 }
 
-const NO_DATA_TEXT = "Sorry, I've not found you anything for that."
-const HELP_TEXT = "Type 'suggest' to see 3 inspiring recipes or type the name of a cuisine, course, ingredient, dish or dietary requirement (e.g. vegan or gluten free) to search for a recipe. Easy!"
+// Not used yet - const FEEDBACK_TEXT = "I'd love to know what you make of my sous chef skills. Could you spare a few moments to give some feedback?"
+// Not used yet - const NOT_SMART_ENOUGH_TEXT = "Sorry, I'm not smart enough for complex commands. Could you rephrase your request?"
+const NO_DATA_TEXT = "Sorry, I couldn't source anything for that. What kind of meal do you fancy eating? For example, you could type 'pasta' or 'breakfast', or try 'chicken curry' or 'vegetarian'."
```
Comment (alastairjardine) [inline index.js]: "Source" feels a little funny. What about "find anything for that"

**Comment by alastairjardine** [comment]
@chriswilk Added a few copy suggestions on the file directly

File: index.js (hunk 3 above)
Comment (chriswilk) [inline index.js]: I think Eleni suggested "source" as a play on words

**Comment by tomrf1** [comment]
@chriswilk I've fixed the build, it wasn't your fault. Go ahead and merge when ready

File: index.js (hunk 3 above)
Comment (tomrf1) [inline index.js]: the problem with puns is that they lose something when you have to write them down!

Comment (alastairjardine) [inline index.js]: OH! Sauce/source. Got it 😂

**Comment by chriswilk** [comment]
@tomrf1 Can you check this again?
